index.d.ts

```diff
@@ -1,107 +0,0 @@
-// TODO:  don't have the defs from the hooks PR yet, fix later
```

Comment:
Is this moved to the https://github.com/feathersjs-ecosystem/feathers-typescript repo? Just curious.

Reply:
Yeah, will be. I don't want to mess with our repos because TypeScript definitions are broken all the time. They'll be published separately and then pulled in.